easybuild/easyconfigs/p/ParallelIO/ParallelIO-2.2.2a-intel-2017a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'ParallelIO'
+version = '2.2.2a'
+
+homepage = 'https://github.com/NCAR/ParallelIO'
+description = """A high-level Parallel I/O Library for structured grid applications"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+sources = ['pio%s.tar.gz' % '_'.join(version.split('.'))]
+source_urls = ['https://github.com/NCAR/ParallelIO/archive/']
+checksums = ['052a03e8e3b5ac244dc53b397a9920af3066efffa216ea7f8e5e4ac73a8b8033']
+
+separate_build_dir = True
+
+dependencies = [
+    ('PnetCDF', '1.8.1'), 
+    ('netCDF-Fortran', '4.4.4'), 
+]
+
+builddependencies = [
+    ('CMake', '3.8.2', '', ('GCCcore', '6.3.0')), 
+]
+
+configopts = "-DPIO_ENABLE_TIMING=OFF "
+
+sanity_check_paths = {
+    'files': ['lib/piof.a'], 
+    'dirs': [('include','lib')], 
```

Comment:
This probably doesn't do what you think it does... A tuple `(..., ...)` indicates options, so the way you have it now, the sanity check will pass if either a non-empty `include` or `lib` directory is found.

If you want to check for *both* `include` and `lib`, use `['include', 'lib']` .

In this case, just `['include']` suffices though, since you're already checking for a file in `lib`, so the `lib` directory is already guaranteed to be there & non-empty.